content/client-server-api/_index.md

```diff
@@ -2540,6 +2540,9 @@ return a standard error response of the form:
 The `retry_after_ms` key SHOULD be included to tell the client how long
 they have to wait in milliseconds before they can try again.
 
+The `Retry-After` HTTP header MAY be specified by the server. When this header
+is set, it SHOULD be preferred by clients over the `retry_after_ms` value.
```

Comment:
Final point: please could you add a paragraph starting `{{% changed-in v="1.10" %}}` (or possibly `added-in`, if you give it its own section) to document when this changed.

Reply:
Done.